src/core/load_balancing/subchannel_interface.h

```diff
@@ -102,6 +103,8 @@ class SubchannelInterface : public DualRefCounted<SubchannelInterface> {
   // make this API public.
   virtual void CancelDataWatcher(DataWatcherInterface* watcher) = 0;
 
+  virtual std::string address_string() const = 0;
```

Comment:
Let's call this method `address()`.

Also, in the interest of moving in the direction of #34172, let's have this return the address in URI format.  And please document that that's the expected return value.

Reply:
done